README.md

```diff
@@ -39,14 +39,14 @@ If you are developing on a remote machine and want to view the demo in your brow
 * Then you can access the demo interface at `http://your-remote-server.com:{port}`.
 
 ## How can I test my changes in an internal repository without publishing?
+#### Note: If you're using docker, then you should do this outside of docker.
```

Comment:
Changes look good, but just curious why you made these changes.

Reply:
I tried to follow these steps, but they were incorrect and misleading. That's why I thought it's was a good time to introduces the fixes.